coops-ui/src/views/Dashboard.vue

```diff
@@ -181,6 +247,7 @@ export default {
 
   watch: {
     historyFilings () {
+      this.checkPendingFilings(this.historyFilings)
```

Comment:
We could add a conditional but i'm not sure the value. It is additive and not destructive, possibly useable with future entities as well. If you feel it could impact performance, i could certainly add it in. Though which conditional is debatable, i was informed that in most instances COOPs will be the exception when compared with other entity types, whereas BCOMPs will be more similar. Thoughts on this?

Reply:
I think performance isn't an issue; this will only be run once after all.

I'm just concerned about the possibility of accidentally applying (or appearing to apply) business rules to non-BCorp businesses. In the code one cannot tell that this applies only to BCorps. A conditional would clarify this, but a comment might suffice.

As for "if (BCorp)" vs "if (!Coop)", I prefer the first since today we know this applies only to BComps (instead of assuming it applies to all but Coops).

Reply:
Sounds good, done!

Reply:
Comment is fine, thanks.